Fixes undefined symbol error on newest nightly

Elekrisk opened the pull request:
This is a horrible fix, but as the name of the 'missing' function is rust_oom,
it should hopefully never be called,
as we have full control over the allocator :)

Elekrisk committed:
Fixed undefined symbol error on newest nightly

This is a horrible fix, but as the name of the 'missing' function is rust_oom,
it should hopefully never be called,
as we have full control over the allocator :)

brynolv reviewed (approved):
Ahaha, this is horrible 🤣

brynolv commented:
Maybe run `fmt` though 😉

Elekrisk committed:
Manually format

The `rustfmt` component is not available on the latest nightly,
so couldn't check beforehand

Vikdos reviewed (approved):
Using a specific nightly version would be better, for example 2021-05-08.
But ok.
